File: cp/seq.cpp
#include<iostream.h>
using namespace std;
int main(){
    int odd = 1;
    int even = 1;
    int n;
    cin>>n;
    for(int i = 0 ; i<=n; i++){
        if(i % 2 == 0){
            odd *= 3;
        }else{
            even *= 2;
        }
    }

    if(n % 2 == 0){
        printf("%d", odd);
    }else{
        printf("%d", even);
    }
    return 0;
}

File: cp/seq.py
def series(n):
    ans = [0 for _ in range(n)]
    even = 1
    odd = 1
    for i in range(0 ,n-1):
        if(i % 2 == 0):
            odd *= 3
        else:
            even *= 2
    if n%2 == 0:
        return odd
    else:
        return even






if __name__ =='__main__':
    n = int(input())
    print(series(n))

